routes/user/user.js

```diff
@@ -247,3 +247,19 @@ exports.activateUser = function(req, res, next) {
             }
         });
 };
+
+exports.updateUser = function(req, res, next) {
```

Comment:
@thinkking What is the endpoint used to access this function?  Also we shouldn't be retrieving using the token.  Try finding by user id.  I suggest: ```PUT /user/:id ```

In HTTP web services PUT is seen as an update to the resource.